docs/source/azure/configuration/acitutorial.rst

```diff
@@ -0,0 +1,142 @@
+.. _azure_configuration_acitutorial:
```

Comment:
This reads very well. I will go through it step by step tomorrow to make sure that each step works